mail/common/src/test/java/com/fsck/k9/mail/BoundaryGeneratorTest.kt

```diff
@@ -19,9 +19,10 @@ class BoundaryGeneratorTest {
 
     @Test
     fun generateBoundary() {
-        val random = createRandom(
-            0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 35,
-        )
+        val seed = IntRange(0, 28).toList().toIntArray()
+
+        val random = createRandom(*seed, 35)
+
```

Comment:
Note (for future changes): I think it's useful when tests are separated into arrange, act, assert blocks. Most of the time this separation can be easily achieved by using an empty line between the blocks. By introducing additional empty lines, this change makes it somewhat harder to figure out at a glance which lines belong to which block.